app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/MainScaffold.kt

```diff
@@ -65,6 +72,16 @@ private fun MainScaffold(
     currentDestination: MainNavigation,
     navController: NavHostController,
     content: @Composable () -> Unit,
+) {
+    Navigation(currentDestination, navController::navigateToSelectedItem, navType, content)
+}
```

Comment:
Why create a method to call another method ?

Reply:
Fixed in https://github.com/Infomaniak/android-SwissTransfer/pull/10#discussion_r1694758215